motr/io_nw_xfer.c

```diff
@@ -1724,14 +1724,6 @@ static void nw_xfer_req_complete(struct nw_xfer_request *xfer, bool rmw)
 			       item, item->ri_type->rit_opcode, item->ri_error,
 			       item->ri_sm.sm_state);
 
-			/* Maintains only the first error encountered. */
```

Comment:
Why is this removed?

So how is the rpc item error populated to "xfer error" and then to "ti" error?

Reply:
It's set at `io_bottom_half()`, here: https://github.com/Seagate/cortx-motr/blob/3b70300f63ebd55fb477126b71b01f43326aaa97/motr/io_req_fop.c#L344-L354